rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxToolbar.java

```diff
@@ -27,6 +28,35 @@
     return Observable.create(new ToolbarItemClickOnSubscribe(view));
   }
 
+  /**
+   * Create an observable which emits on {@code view} navigation click events. The emitted value is
+   * unspecified and should only be used as notification.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link Toolbar#setNavigationOnClickListener}
+   * to observe clicks. Only one observable can be used for a view at a time.
+   */
+  @CheckResult @NonNull
+  public static Observable<Object> navigationClicks(@NonNull Toolbar view) {
+    return Observable.create(new ToolbarNavigationClickOnSubscribe(view));
+  }
+
+  /**
+   * Create an observable of click events for {@code view}'s navigation clicks.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link Toolbar#setNavigationOnClickListener}
+   * to observe clicks. Only one observable can be used for a view at a time.
+   */
+  @CheckResult @NonNull
+  public static Observable<ViewClickEvent> navigationClickEvents(@NonNull Toolbar view) {
```

Comment:
I don't have any usecase either, just added it to be in line with the normal click bindings, because it uses the `View.OnClickListener`.